meta-lmp-base/recipes-devtools/python/python3-func-timeout_4.3.5.bb

```diff
@@ -1,7 +1,7 @@
 SUMMARY = "Support running any existing function with a given timeout"
 HOMEPAGE = "https://github.com/kata198/func_timeout"
 SECTION = "devel/python"
-LICENSE = "LGPLv3 & LGPL-2.0"
+LICENSE = "LGPL-3.0-only & LGPL-2.0-only"
```

Comment:
Looking at the license again, I believe the correct one to use here is just LGPL-3.0-only. Based on https://github.com/kata198/func_timeout/blob/master/LICENSE.